backend/data/src/main/kotlin/io/tolgee/service/EmailVerificationService.kt

```diff
@@ -63,6 +66,49 @@ class EmailVerificationService(
     return null
   }
 
+  @Transactional
+  fun resendEmailVerification(
```

Comment:
I think, maybe strict `@RateLimited` annotation on the endpoints would be enough and simplier.

Reply:
Tell me if I am wrong.

Reply:
I think you're right, but I wanted to solve the problem with incorrect email(more in conversation below).